roles/neutron-common/defaults/main.yml

```diff
@@ -79,6 +80,36 @@ neutron:
     service_plugin:
       - neutron_lbaas.services.loadbalancer.plugin.LoadBalancerPluginv2
     service_provider: LOADBALANCERV2:Haproxy:neutron_lbaas.drivers.haproxy.plugin_driver.HaproxyOnHostPluginDriver:default
+    f5:
+      enabled: False
+      debug: False
+      periodic_interval: 10
+      f5_ha_type: standalone
+      f5_external_physical_mappings: "default:1.2:True"
+      f5_vtep_folder: 'Common'
+      f5_vtep_selfip_name: 'vtep'
+      advertised_tunnel_types: vxlan
+      f5_populate_static_arp: False
+      l2_population: True
+      f5_global_routed_mode: False
+      use_namespaces: True
+      max_namespaces_per_tenant: 1
+      f5_route_domain_strictness: False
+      f5_snat_mode: True
+      f5_snat_addresses_per_subnet: 1
+      f5_common_external_networks: False
+      f5_bigip_lbaas_device_driver: f5_openstack_agent.lbaasv2.drivers.bigip.icontrol_driver.iControlDriver
+      icontrol_hostname: 256.256.256.256
+      icontrol_username: admin
+      icontrol_password: admin
+      f5_parent_ssl_profile: clientssl
+      service_provider: LOADBALANCERV2:F5Networks:neutron_lbaas.drivers.f5.driver_v2.F5LBaaSV2Driver:default
+      f5_service_provider: "https://github.com/F5Networks/neutron-lbaas/releases/download/v9.1.0/f5.tgz"
+      python_dependencies:
+        - { name: "git+https://github.com/F5Networks/f5-openstack-lbaasv2-driver@v9.3.0.b2" }
```

Comment:
As per installing f5 driver/agent all u need to do:
1: neutron-common/defaults/main.yml - add f5 driver/agent to python_dependencies
https://github.com/blueboxgroup/ursula/blob/master/roles/neutron-common/defaults/main.yml#L99
2: add f5-oslbaasv2-agent  to alternatives
3: I will cover giftwrap side when f5 lbaasv2 driver/agent is released so that it is in neutron giftwrap

The breakout of haproxy and f5 in it;s own task file is good. Just need to remove lot of code as above method simply how u install ad it will be always part of neutron venv for giftwrap and for source install is handle by openstack-source role part of install neutron.